blueocean-dashboard/src/main/js/components/PipelinePage.jsx

```diff
@@ -32,6 +34,15 @@ export function pipelineBranchesUnsupported(pipeline) {
     return false;
 }
 
+const classicConfigLink = (pipeline) => {
+    let link = null;
+    if (AppConfig.getInitialUser() !== 'anonymous') {
+        link = <a href={buildClassicConfigUrl(pipeline)} target="_blank"><Icon size={24} icon="settings" style={{ fill: '#fff' }} /></a>;
```

Comment:
Please add the following to the ```a```

```
rel="noopener noreferrer"
```

https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/